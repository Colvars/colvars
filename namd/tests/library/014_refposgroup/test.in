colvarsTrajFrequency    1


colvar {
    name r

    outputAppliedForce yes

    rmsd {
        refPositionsFile ../Common/310turn-ideal.xyz
        debugGradients yes
        atoms {
            atomNumbers  4
            rotateToReference yes
            centerToReference yes
            fittingGroup {
                atomNumbers  54 64 74 84 99 
            }
            refPositionsFile ../Common/310turn-ideal.xyz # ref prot positions for fitting
        }
    }
}

harmonic {
    colvars r
    centers 0.0
    forceConstant 0.1
}

