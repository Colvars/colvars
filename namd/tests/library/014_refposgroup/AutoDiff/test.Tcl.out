TCL: TOTAL FORCE: ( 1.384807019055856e-9 3.878058707496024e-10 9.441096737727861e-9 )
TCL: TOTAL TORQUE: ( 9.617198148959005e-9 -6.671102603661438e-8 1.3296069245605224e-9 )
TCL: TOTAL FORCE: ( 1.3845898871878148e-9 3.876407805858406e-10 9.440526915760472e-9 )
TCL: TOTAL TORQUE: ( 9.58525481209449e-9 -6.669777796730614e-8 1.3328811387935957e-9 )
TCL: TOTAL FORCE: ( 1.384747011501375e-9 3.876299281557749e-10 9.440897230650336e-9 )
TCL: TOTAL TORQUE: ( 9.551191837431361e-9 -6.668942509335807e-8 1.3372550289325602e-9 )
TCL: TOTAL FORCE: ( 1.3847027136026924e-9 3.8810379909826054e-10 9.441948223276597e-9 )
TCL: TOTAL TORQUE: ( 9.519129040569396e-9 -6.668528440556543e-8 1.3450211500121156e-9 )
TCL: TOTAL FORCE: ( 1.3846692126229243e-9 3.8859587769835e-10 9.44301592475938e-9 )
TCL: TOTAL TORQUE: ( 9.487008512110151e-9 -6.668641860940738e-8 1.3531361586771595e-9 )
TCL: TOTAL FORCE: ( 1.3843895196874456e-9 3.892627331580911e-10 9.443209769699479e-9 )
TCL: TOTAL TORQUE: ( 9.455814797831863e-9 -6.668312035884583e-8 1.3625308659115376e-9 )
TCL: TOTAL FORCE: ( 1.3839312196228803e-9 3.896173383921564e-10 9.442643722490374e-9 )
TCL: TOTAL TORQUE: ( 9.423963831522997e-9 -6.66716588604288e-8 1.3697779022159295e-9 )
TCL: TOTAL FORCE: ( 1.3834813572533022e-9 3.8981154415473895e-10 9.441210035987524e-9 )
TCL: TOTAL TORQUE: ( 9.39311961545286e-9 -6.665373897263294e-8 1.375589642194086e-9 )
TCL: TOTAL FORCE: ( 1.383253483977498e-9 3.8989927952925996e-10 9.43917471962763e-9 )
TCL: TOTAL TORQUE: ( 9.36619892755175e-9 -6.663279394913957e-8 1.3798079900873006e-9 )
TCL: TOTAL FORCE: ( 1.383288567025076e-9 3.903983525344046e-10 9.43607558756554e-9 )
TCL: TOTAL TORQUE: ( 9.336882378363498e-9 -6.660509610512122e-8 1.3869025372592603e-9 )
TCL: TOTAL FORCE: ( 1.3836645162967898e-9 3.910716472876885e-10 9.431551373229041e-9 )
TCL: TOTAL TORQUE: ( 9.307735915342619e-9 -6.656929452120153e-8 1.3947462074170858e-9 )
TCL: TOTAL FORCE: ( 1.383956810263598e-9 3.916388602309695e-10 9.426974978410385e-9 )
TCL: TOTAL TORQUE: ( 9.281833523999694e-9 -6.653200479433963e-8 1.4013895599518378e-9 )
TCL: TOTAL FORCE: ( 1.3839795143244515e-9 3.9211683899864624e-10 9.422767954792022e-9 )
TCL: TOTAL TORQUE: ( 9.256830413306716e-9 -6.64963777374794e-8 1.4075592358331335e-9 )
TCL: TOTAL FORCE: ( 1.3838287182821318e-9 3.9261055517769705e-10 9.417791990706803e-9 )
TCL: TOTAL TORQUE: ( 9.234642384114977e-9 -6.64554629103975e-8 1.4134880488292367e-9 )
TCL: TOTAL FORCE: ( 1.3835778911452934e-9 3.927678460247108e-10 9.412744972348008e-9 )
TCL: TOTAL TORQUE: ( 9.216622132157681e-9 -6.640994820727997e-8 1.4163497041863593e-9 )
TCL: TOTAL FORCE: ( 1.3833555689846122e-9 3.9301065180019634e-10 9.407115364457042e-9 )
TCL: TOTAL TORQUE: ( 9.198239947494358e-9 -6.635960181355927e-8 1.4197321096531823e-9 )
TCL: TOTAL FORCE: ( 1.3832198719754274e-9 3.9355749215097546e-10 9.401293632471663e-9 )
TCL: TOTAL TORQUE: ( 9.17791531662715e-9 -6.630868121249023e-8 1.425466855664581e-9 )
TCL: TOTAL FORCE: ( 1.3833443557320635e-9 3.940626436271799e-10 9.394919009420022e-9 )
TCL: TOTAL TORQUE: ( 9.155735281041189e-9 -6.625382553693271e-8 1.4308377815908102e-9 )
TCL: TOTAL FORCE: ( 1.383406056376657e-9 3.941313664324042e-10 9.38953176321533e-9 )
TCL: TOTAL TORQUE: ( 9.135093126388938e-9 -6.620808390422894e-8 1.4332044995235549e-9 )
TCL: TOTAL FORCE: ( 1.3834732248696469e-9 3.936540260429666e-10 9.384015065005968e-9 )
TCL: TOTAL TORQUE: ( 9.11666298009095e-9 -6.615982339752691e-8 1.4313049634395725e-9 )
